legal-api/src/legal_api/core/meta/filing.py

```diff
@@ -254,6 +254,22 @@ class FilingTitles(str, Enum):
             {'types': 'BC,BEN,CC,ULC', 'outputs': ['letterOfConsent', ]},
         ]
     },
+    'continuationIn': {
+        'name': 'continuationIn',
+        'title': 'Continuation In',
+        'displayName': {
+            'BC': 'BC Limited Company Continuation In Application',
+            'BEN': 'BC Benefit Company Continuation In Application',
+            'CC': 'BC Community Contribution Company Continuation In Application',
+            'ULC': 'BC Unlimited Liability Company Continuation In Application',
+        },
+        'codes': {
+            'BC': 'CONTI',
+            'BEN': 'CONTI',
+            'ULC': 'CONTI',
+            'CC': 'CONTI'
```

Comment:
At what point do we start using the new legal types (C, CBEN, CUL, CCC)?

Reply:
I will update it in my ticket